src/backend/utils/misc/guc.c

```diff
@@ -8200,7 +8200,7 @@ static struct config_string ConfigureNamesString[] =
 			NULL
 		},
 		&standby_addr_host,
-		"localhost", NULL, NULL
+		"none", NULL, NULL
```

Comment:
https://issues.apache.org/jira/browse/HAWQ-1266 this jira is created for discussion. Thanks @paul-guo- 's thorough think.